ffcx/ir/elementtables.py

```diff
@@ -207,6 +205,19 @@ def get_ffcx_table_values(points, cell, integral_type, ufl_element, avg, entityt
 
         component_element = fiat_element.elements()[component_element_index]
 
+        # Get the block size to switch XXYYZZ ordering to XYZXYZ
+        if isinstance(ufl_element, ufl.VectorElement) or isinstance(ufl_element, ufl.TensorElement):
+            block_size = len(fiat_element.elements())
```

Comment:
FIAT element has num_sub_element, you are calling this somewhere above, would be better to be consistent.